### DISCUSSION TITLE
Trying to pull commit done in a repo present inside an organisation. Getting {     "message": "Not Found",     "documentation_url": "https://docs.github.com/rest/reference/repos#list-commits" }

### DISCUSSION THREAD

**Discussion by izrake** [octokit/octokit.js#1886] [Q&A]
I using octokit to perform action.

**Steps to reproduce:**

I am making a call using octo
`
  const response = await octokit.request("GET /repos/{owner}/{repo}/commits", {
            owner: 'izrake',
            repo: 'argo-react'
        });
`
argo-api is my repo and also it is an open source.

After making this call i am getting an error 
But when i am making a call to the repos which are directly reside to user level then i am successfully able to retrieve the data.
Is there any other way of getting the commits of repos which belongs to an org using octo ?

Note: I have configured proper scopes for access_token

Any help will be highly appreciated. If any additional info is required please let me know i will be happy to share.

**Comment by gr2m** [comment]
The 404 usually means that the authentication is incorrect. 

I tried to reproduce the problem you mentioned but it works as expected: https://runkit.com/gr2m/octokit-rest.js-1886/1.0.0

You say the problem would be for a repository in an organization though? Did you create a personal access token at https://github.com/settings/tokens/new or did you create the token using an OAuth/GitHub App?

**Reply by izrake** [reply]
I have an app which creates access_token on behalf of user. Using that token I was making the call. With that token i am able to see all the private repos but while fetching the commits i was getting the error.

**Reply by izrake** [reply]
@gr2m any idea? Do I need to only use a user-created token or I can use the OAuth access token

**Reply by gr2m** [reply]
Tokens created by GitHub Apps are constraint to the permissions of the app. If the app does not have the permission, then the user tokens don't have it either.

Could you create a minimal test case which includes the GitHub App code to create the user token?
